src/main/java/nextstep/ladder/application/RandomCreatePoint.java

```diff
@@ -0,0 +1,21 @@
+package nextstep.ladder.application;
+
+import java.util.Random;
+
+public class RandomCreatePoint implements CreatePointStrategy{
+    private static RandomCreatePoint randomCreatePoint = new RandomCreatePoint();
```

Comment:
```suggestion
    private static final RandomCreatePoint randomCreatePoint = new RandomCreatePoint();
```

클래스 변수의 경우 final 키워드를 붙여주는 것이 관례인데 왜 이러한 관례가 생겼는지 고민해보아요